### PR TITLE
Create creating-marketplace-kit-logs.liquid

### PR DESCRIPTION
**Pull request by diana-lakatos**
Creating Marketplace Kit Logs tutorial for the How-To Guides / QA section.

### CODE REVIEW

File: marketplace_builder/views/pages/how-to/qa/creating-marketplace-kit-logs.liquid [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+metadata:
+  title: Creating Marketplace Kit Logs
+  description: This guide will help you create your own Marketplace Kit logs using the `log` Liquid tag.
+slug: how-to/qa/creating-marketplace-kit-logs
+searchable: true
+---
+
+This guide will help you create your own Marketplace Kit logs using the `log` Liquid tag. This tag allows you to print any information to Marketplace Kit logs as type info, debug, or error, and list them in your console next to your sync. 
+
+You can use the `log` Liquid in any file that accepts Liquid, so you can use them in pages, layouts, partials, Authorization Policies, async callbacks, etc.
```
Comment (pavelloz) [inline marketplace_builder/views/pages/how-to/qa/creating-marketplace-kit-logs.liquid]: You can use the `log` *tag*

Comment (pavelloz) [reply marketplace_builder/views/pages/how-to/qa/creating-marketplace-kit-logs.liquid]: Or `log Liquid tag` as below

**Comment by diana-lakatos** [comment]
@pavelloz Thank you! Fixed.

**Comment by pavelloz** [comment]
:)

**Pull request merged**